dashboard/src/types/tree/Tree.tsx

```diff
@@ -90,9 +90,29 @@ const origins = [
   'syzbot',
   'tuxsuite',
 ] as const;
+
+const paths = [
+  '',
+  '/',
+  '/tree',
+  '/devices',
+  '/labs',
+  '/tree/$treeId',
+  '/tree/$treeId/test',
+  '/tree/$treeId/test/$testId',
+  './',
+  '../',
+  '/tree/$treeId/build/$buildId',
+] as const;
```

Comment:
These paths are only for the `SideMenu` so it should be in that file, and you can delete all the paths that are not a SideMenu lilnk